src/renderer/components/ImagePicker/ImagePicker.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
+import './ImagePicker.scss';
+
+interface Props {
+  image?: string;
+  onClick?: () => void;
+  error?: boolean;
+}
+
+const ImagePicker: React.FC<Props> = ({ image, error }) => {
+  if (!image) {
+    return error ? (
+      <div className='img error'>
+        <div className='imgplus error' />
+      </div>
+    ) : (
+      <div className='img'>
+        <div className='imgplus' />
+      </div>
+    );
+  } else {
+    return <img className='img selected' src={image} />;
+  }
+};
```

Comment:
I believe the component can be cleaned up a little by refactoring the if statement and using a function to generate the class name based on the `error` value. You can return the `img` if the `image` prop is present and then have default return with the classes generated from a function like so:

```tsx
const ImagePicker: React.FC<Props> = ({ image, error }) => {
  if (image)
    return <img className='img selected' src={image} alt='Example alt' />;

  const makeClass = (base: string): string => (error ? `${base} error` : base);

  return (
    <div className={makeClass('img')}>
      <div className={makeClass('imgplus')} />
    </div>
  );
};
```

The `onClick` prop also needs to be called when the element is pressed so it can be passed to an `onClick` event of the `div`.

```tsx
const ImagePicker: React.FC<Props> = ({ image, error, onClick }) => {
  // ... other code

  return (
    <div
      className={makeClass('img')}
      onClick={onClick}
    >
      <div className={makeClass('imgplus')} />
    </div>
  );
};
```

With the addition of an `onClick` handler comes a couple of things though. The element should define a role if it has event listeners so that needs to be included. It is a good practice to take care of accessibility and therefore have a keyboard event accompany the `onClick`. If that is done then also the element should be focusable. And if it is focusable it should also have title. Therefore, this leads to something along the lines of the below: 

```tsx
const ImagePicker: React.FC<Props> = ({ image, error, onClick }) => {
  // ... other code

  return (
    <div
      aria-label='image picker' // title for the accessible element
      tabIndex={0} // makes the element focusable with keyboard
      role='button' // see possible roles here -> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
      className={makeClass('img')}
      onClick={onClick}
      onKeyPress={({ key }) => key === 'Enter' && onClick()} // allows the user to fire the onClick action with the press of Enter (when focused)
    >
      <div className={makeClass('imgplus')} />
    </div>
  );
};
```